Celeste.Mod.mm/Patches/Input.cs

```diff
@@ -22,6 +23,15 @@ static class patch_Input {
                 mod.OnInputInitialize();
 
             Everest.Events.Input.Initialize();
+
+            //Sets the slight camera movement on the map to the set debug camera movement keys in Everest mod settings
+            Input.MountainAim = new VirtualJoystick(
```

Comment:
This should probably be done before the event is invoked so that mods can rely on the fact that the event is invoked after Everest's input initialization is complete.

Reply:
I'm not sure if I completely understand what you are saying, but I think I fixed it.